publish-gh-pages.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Could we add the following lines to the top of this? (Feel free to remove the comments):

```sh
#!/bin/bash
set -e           # aborts if there are errors
set -u           # errors if you use an undefined variable
set -o pipefail  # errors if a pipe fails
```